File: 2020/src/j3.cpp
# include <bits/stdc++.h>

using namespace std;

int main() {
  int n;
  cin >> n;
  int x_min=101, x_max=-1, y_min=101, y_max=-1;
  string coord;
  int x, y;
  for (int i = 0; i < n; i++) {
    cin >> coord;
    auto pos = coord.find(",");
    x = stoi(coord.substr(0, pos));
    y = stoi(coord.substr(pos+1));
    x_min = min(x_min, x);
    x_max = max(x_max, x);
    y_min = min(y_min, y);
    y_max = max(y_max, y);
  }

  cout << (x_min-1) << "," << (y_min-1) << endl;
  cout << (x_max+1) << "," << (y_max+1) << endl;
  return 0;
}

File: 2020/src/j3.py
import collections
import itertools
import functools
import math
import re
import bisect
import random

rint = lambda: int(input())
rstr = lambda: input()
rints = lambda: list(map(int, input().split()))
rstrs = lambda: input().split()
wmat = lambda n, mat, sep: '{}\n{}'.format(n, '\n'.join(sep.join(map(str, row)) for row in mat))
warr = lambda n, arr, sep: '{}\n{}'.format(n, sep.join(map(str, arr)))
wl = lambda sep, *arr: sep.join(map(str, arr))

def main():
    n = rint()
    xmin = ymin = math.inf
    xmax = ymax = -math.inf

    for _ in range(n):
        x, y = map(int, input().split(','))
        xmin = min(xmin, x)
        ymin = min(ymin, y)
        xmax = max(xmax, x)
        ymax = max(ymax, y)
    
    print(wl(',', xmin-1, ymin-1))
    print(wl(',', xmax+1, ymax+1))

if __name__ == '__main__':
    main()
